### ISSUE TITLE
Interface size

### ISSUE THREAD

**Issue by WardMaes** [Healix/Gw2Launcher#66]
Is it possible to add the 'Interface size' to the display options?

**Comment by Healix** [comment]
No. The graphics options Gw2Launcher shows are dynamically generated from what GW2 makes available. If ArenaNet ever remembers that they have this and adds the newer options, Gw2Launcher will automatically show them.
